docs/versioned-plugins/outputs/kafka-v11.4.0.asciidoc

```diff
@@ -0,0 +1,533 @@
+[source,java]
+----------------------------------
+KafkaClient {
+  com.sun.security.auth.module.Krb5LoginModule required
+  useTicketCache=true
+  renewTicket=true
+  serviceName="kafka";
+  };
+----------------------------------
+
+Please note that specifying `jaas_path` and `kerberos_config` in the config file will add these
+to the global JVM system properties. This means if you have multiple Kafka inputs, all of them would be sharing the same
+`jaas_path` and `kerberos_config`. If this is not desirable, you would have to run separate instances of Logstash on
+different JVM instances.
+
+[id="{version}-plugins-{type}s-{plugin}-kerberos_config"]
+===== `kerberos_config`
+
+  * Value type is {logstash-ref}/configuration-file-structure.html#path[path]
+  * There is no default value for this setting.
+
+Optional path to kerberos config file. This is krb5.conf style as detailed in https://web.mit.edu/kerberos/krb5-1.12/doc/admin/conf_files/krb5_conf.html
+
+[id="{version}-plugins-{type}s-{plugin}-key_serializer"]
+===== `key_serializer`
+
+  * Value type is {logstash-ref}/configuration-file-structure.html#string[string]
+  * Default value is `"org.apache.kafka.common.serialization.StringSerializer"`
+
+Serializer class for the key of the message
+
+[id="{version}-plugins-{type}s-{plugin}-linger_ms"]
+===== `linger_ms`
+
+  * Value type is {logstash-ref}/configuration-file-structure.html#number[number]
+  * Default value is `0`
+
+The producer groups together any records that arrive in between request
+transmissions into a single batched request. Normally this occurs only under
+load when records arrive faster than they can be sent out. However in some circumstances
+the client may want to reduce the number of requests even under moderate load.
+This setting accomplishes this by adding a small amount of artificial delay—that is,
+rather than immediately sending out a record the producer will wait for up to the given delay
+to allow other records to be sent so that the sends can be batched together.
+
+[id="{version}-plugins-{type}s-{plugin}-max_request_size"]
+===== `max_request_size`
+
+  * Value type is {logstash-ref}/configuration-file-structure.html#number[number]
+  * Default value is `1048576` (1MB).
+
+The maximum size of a request
+
+[id="{version}-plugins-{type}s-{plugin}-message_headers"]
+===== `message_headers`
+
+  * Value type is {logstash-ref}/configuration-file-structure.html#hash[hash]
+  ** Keys are header names, and must be {logstash-ref}/configuration-file-structure.html#string[string]
+  ** Values are header values, and must be {logstash-ref}/configuration-file-structure.html#string[string]
+  ** Values support interpolation from event field values
+  * There is no default value for this setting.
+
+A map of key value pairs, each corresponding to a header name and its value respectively.
+Example:
+[source,ruby]
+----------------------------------
+    message_header => { "event_timestamp" => "%{@timestamp}" }
```

Comment:
```suggestion
    message_headers=> { "event_timestamp" => "%{@timestamp}" }
```

Reply:
This change isn't required to clear docs-ci. I'm changing it for content accuracy.